mobile/screens/Resource/ResourceHomeScreen.js

```diff
@@ -346,8 +346,10 @@ const ResourceHomeScreen = ({ navigation, props }) => {
     return (
       <Pressable
         onPress={() =>
+        { console.log("we are looking at this item right here", JSON.stringify(item))
           navigation.navigate("ResourceArticle", { resource: item })
         }
```

Comment:
Nit: Can you indent this? Also if the console log is for debug purpose remove it if you don't need it anymore. If you still need it then it's best to follow the current log format. For example, `[ResourceHomeScreen]: <message>` so that when someone read the log they can know where it's coming from

Reply:
I will remove this comment as it primarily for debugging.